### ISSUE TITLE
Value of indent property of syntax profile is copied to expanded JSX

### ISSUE THREAD

**Issue by Gua-naiko-che** [emmetio/emmet#521]
I'm learning to use Emmet, so I'm not sure if this is a bug.

I want to force each tag to be expanded in a new line. After reading [this documentation](https://docs.emmet.io/customization/syntax-profiles/), I added this to my Visual Studio Code setting file, and it worked as desired:
```
    "emmet.syntaxProfiles": {
        "jsx": {
            "inline_break": 1
        }
    }
```
However, during my experiments, I found that if I add the `indent` property to the syntax profile, the value of the property gets copied to the expanded code!

## Steps to reproduce:
1. Use Visual Studio Code 1.18.1
2. Put this in the user settings file:
```
    "emmet.syntaxProfiles": {
        "jsx": {
            "inline_break": 1,
            "indent":"whatever"
        }
```
3. Create a *.tsx file (Typescript React)
4. Insert this code and hit TAB: `div.buttons>button.something`
5. It is expanded to the following:
```
      <div className="buttons">
      whatever<button className="something"></button>
      </div>
```

**Comment by sergeche** [comment]
`indent` is a string that is used for indentation, most likely it should be a tab character or a number of spaces

**Comment by Gua-naiko-che** [comment]
From [the documentation](https://docs.emmet.io/customization/syntax-profiles/):

> indent: indent tags on new lines, boolean.

**Comment by sergeche** [comment]
These docs are for old Emmet version, VS Code uses new, upcoming release of Emmet 2.0

**Comment by Gua-naiko-che** [comment]
Ah! Ok, thanks!
I got confused because the [Visual Studio Code documentation](https://code.visualstudio.com/docs/editor/emmet#_emmet-abbreviations-in-other-file-types) points to the old documentation.

So for forcing each tag to be expanded in a new line, should I use this?
```
"emmet.syntaxProfiles": {
    "jsx": {
        "inline_break": 1
    }
}
```
Or is there a better way of doing it? I tried using `tag_nl`, but it didn't seem to produce any effect...
